app/poros/trip.rb

```diff
@@ -1,55 +1,55 @@
-class Trip
-  attr_reader :id,
-              :origin_city,
-              :destination_city,
-              :origin_iata,
-              :destination_iata,
-              :latitude,
-              :longitude,
-              :price,
-              :departure_datetime,
-              :arrival_datetime,
-              :r_departure_datetime,
-              :r_arrival_datetime,
-              :booking_link,
-              :trip_duration,
-              :flight_id,
-              :min_f,
-              :max_f,
-              :min_c,
-              :max_c,
-              :day_feels_like_c,
-              :day_feels_like_f,
-              :description,
-              :weather_date,
-              :weather,
-              :sky_coverage
+# class Trip
```

Comment:
Why do we want our poros to be commented out?

Reply:
Because a `Trip` model has been created I think a PORO is unnecessary now.